### PR TITLE
fix: make styling consistent

### PR DESCRIPTION
**Pull request by ch-rfishwick**
A few other changes required in order to make styling consistent with prototype.

### CODE REVIEW

File: server/views/applications/add_key.njk [hunk 1]
```diff
@@ -66,51 +64,50 @@
...
-    <textarea class="govuk-textarea{{ ' govuk-input--error' if this_errors.javaScriptDomain is not undefined }}" id="more-detail-domain" name="javaScriptDomains" rows="5" aria-describedby="more-detail-hint-domain">{{ this_data.body.javaScriptDomains }}</textarea>
     {% if this_errors.javaScriptDomain is not undefined %}
       <span id="err-id-terms" class="govuk-error-message">
         <span class="govuk-visually-hidden">Error:</span> {{ this_errors.javaScriptDomain.inline }}
       </span>
     {% endif %}
+    <textarea class="govuk-textarea{{ ' govuk-input--error' if this_errors.javaScriptDomain is not undefined }}" id="more-detail-domain" name="javaScriptDomains" rows="5" aria-describedby="more-detail-hint-domain">{{ this_data.body.javaScriptDomains }}</textarea>
   </div>
-  <div class="govuk-form-group" id='redirect-uris' {{'' if  this_data.body.keyType === 'web' else 'style=display:none;' }}>
+  <div class="govuk-form-group{{' govuk-form-group--error' if this_errors.redirectURIs is not undefined }}" id='redirect-uris' {{'' if  this_data.body.keyType === 'web' else 'style=display:none;' }}>
```
Comment (CH-JonathanBlake) [inline server/views/applications/add_key.njk:97]: happens on a few locations, no space after the {{ opens.

Comment (ch-rfishwick) [reply server/views/applications/add_key.njk]: changed

**Review by CH-JonathanBlake — APPROVED** [review]

File: server/views/applications/add_key.njk (hunk 1 above)
Comment (ch-rfishwick) [reply server/views/applications/add_key.njk]: changed some more

Comment (Zain-Abbas1) [reply server/views/applications/add_key.njk]: Doesn't look like all have been changed yet

**Comment by ch-rfishwick** [comment]
You mean the ‘ govuk-form-group—error’? I think your github view might be out of date. I can’t see the comment on github….

**Review by Zain-Abbas1 — APPROVED** [review]

**Pull request merged**